### ISSUE TITLE
Duplo train

### ISSUE THREAD

**Issue by anthonyangel** [pybricks/support#182]
**Question**
Do you support the duplo train hub?

**Context**
Does this project support the Duplo train which comes in set 10874 / 10875?

**Comment by dlech** [comment]
No, we don't support it. But we aren't opposed to supporting it eventually. The more popular hubs have higher priority though.

**Comment by laurensvalk** [comment]
@anthonyangel - do you have such a Duplo train hub already? If so:

1. Did you ever see the app update the firmware on the hub? If so, it would have been most likely when you used the app for the first time.
2. What happens if you follow the first 5 seconds of the steps in [this video](https://pybricks.com/wp-content/uploads/2020/06/install.mp4) from [this page](https://docs.pybricks.com/en/latest/start_pup.html)?
   - In other words, does a light start blinking when you press and hold the button?
   - And if so, do you see "LEGO Bootloader" as in the video? That would be a good sign. Do **not** click pair, as we don't have a firmware for that hub yet. Nothing bad should happen though, but just in case...

**Comment by anthonyangel** [comment]
I can't remember ever seeing a firmware update from the app.

I tried to pair, as per the video - the hub only turns on (lights come on & it makes a noise) when the green power button is released, and nothing was showing in the pairing menu

**Comment by laurensvalk** [comment]
Thanks, that is good to know. It is quite possible that this hub is not upgradeable.

**Comment by dlech** [comment]
This hub doesn't have upgradeable firmware but can be remote controlled using the `LWP3Device` class.

**Comment by laurensvalk** [comment]
I've made an example project for this. Still needs to be published though. We might keep the issue open for that.

**Comment by FideliusFalcon** [comment]
I tried the [example code](https://pybricks.com/projects/tutorials/wireless/hub-to-hub/pybricks-to-official-fw/), but it's printing "Connected!" right away, and doens't really connect. Is there some method to debug this? 
There's also a type error regarding the color object:
```
TypeError: unhashable type: 'Color'
```
